templates/applications/application.html

```diff
@@ -4,136 +4,311 @@
 <meta name="robots" content="noindex">
 {% endblock %}
 
-{% block careers_content %}
+{% block content %}
+<script>
+  function showProgressDetail(stage) {
+  const showMoreBtn = document.querySelector(`.show-more-${stage}`);
+  const detail = document.querySelector(`.progress-detail-${stage}`)
+  if ( detail.classList.contains("u-hide")) {
+    detail.classList.remove("u-hide")
+    showMoreBtn.innerHTML="Show less"
+  } else {
+    detail.classList.add("u-hide")
+    showMoreBtn.innerHTML="Show more"
+  }
+}
+</script>
 {% if application["status"] == "active" %}
-<section class="p-strip--suru-topped u-no-padding--bottom">
-    <div class="row">
-        <h1>Thanks for applying {{ candidate["first_name"] }}!</h1>
-        {% if application['source']['id'] == 2 %}
-            <p>You applied for the role: <a href="/careers/{{ application['job_post_id'] }}"><strong>{{ application["jobs"][0]["name"] }}</strong></a></p>
-        {% else %}
-            <p>You applied for the role: <strong>{{ application["jobs"][0]["name"] }}</strong></p>
+  <section class="p-strip--light is-deep">
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <h1 class="u-sv2">Thanks for applying, {{ candidate["first_name"] }}.</h1>
+        {% if "stage_progress" in application and application["stage_progress"]["assessment"] and not application["stage_progress"]["early_stage"]  %}
+            {% if application['source']['id'] == 2 %}
+              <p class="u-no-margin--bottom">We have received your application for the role <a href="/careers/{{ application['job_post_id'] }}">{{ application["jobs"][0]["name"] }}</a></p>
+            {% else %}
+              <p class="u-no-margin--bottom">We have received your application for the role {{ application["jobs"][0]["name"] }}.</p>
+            {% endif %}
+            <p>Your hiring lead is {{ hiring_lead["name"] }}.</p>
+        {% elif "stage_progress" in application and application["stage_progress"]["early_stage"] %}
+            {% if application['source']['id'] == 2 %}
+              <p>We have received your application for the role <a href="/careers/{{ application['job_post_id'] }}">{{ application["jobs"][0]["name"] }}</a></p>
+            {% else %}
+              <p>We have received your application for the role {{ application["jobs"][0]["name"] }}.</p>
+            {% endif %}
+            {% if candidate["application_ids"]|length >= 2 %}
+              <p class="u-no-margin--bottom">STATUS OF YOUR OTHER APPLICATIONS:</p>
```

Comment:
The font size for the "status..." bit looks a bit different on figma, but I'm not sure if it's just me? There's also a bit of whitespace missing before the application(s) list
<img width="679" alt="Screenshot 2022-08-15 at 13 04 19" src="https://user-images.githubusercontent.com/17607612/184624755-21850c8f-ae5b-45f3-b697-2ac48523b6ab.png">

Reply:
I added `p-heading--5 u-no-padding--top u-no-margin--bottom u-sv1` classes to the "STATUS..". Many classes but would it look better now?